host/sgx/loadelf.c

```diff
@@ -50,6 +52,8 @@ static oe_result_t _unload_image(oe_enclave_image_t* image)
     if (image)
     {
         _unload_elf_image(&image->elf);
+        if (image->module)
```

Comment:
Should we unload the module first and then the image?

Reply:
This should be fine as the base image (referenced by image->elf) and the module (image->module) are loaded separately. We just need to make sure unloading both of them before memset the image itself (line 57).